src/plugins/creditor/declaration.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType, EdgeType} from "../../analysis/types";
+import {NodeAddress, EdgeAddress} from "../../core/graph";
+
+export const nodePrefix = NodeAddress.fromParts(["sourcecred", "creditor"]);
+export const edgePrefix = EdgeAddress.fromParts(["sourcecred", "creditor"]);
+
+export const CreditorEntryType: NodeType = deepFreeze({
+  name: "CredNode",
+  pluralName: "CredNodes",
+  prefix: NodeAddress.append(nodePrefix, "credNode"),
+  defaultWeight: 0,
+  description:
+    "An creditor node, describing a scoped improvement to a project from proposal to completion. \
+    Creditor nodes can be nested  arbitrarily (but non-circularly) to represent complet projects and scopes",
+});
+
+/*
+  Note on the forward and backward naming convention.
+  It follows the core/graph.js documentation to use
+  a <subject> <verb> <object> format to figure out
+  the directionality.
+*/
+
+/**
+ * A contributor (src) CONTRIBUTES TO (verb) an Creditor node (dst).
+ * Forward: a contributor towards the entry node has a small endorsement of that
+ * contribution.
+ * Backward: flows the value of the contribution to the contributors.
```

Comment:
I think we should start with a really generic creditor connection for connecting arbitrary Creditor Nodes, and then later add the specific case where we're connecting to a contribution.

Otherwise most potential connections are inexpressible.